This patch is part of a series of patches for speech-tools for
Debian GNU/Linux, to comply with Filesystem Hierarchy Standard

Index: speech-tools-2.0.95~beta/utils/EST_cutils.c
===================================================================
--- speech-tools-2.0.95~beta.orig/utils/EST_cutils.c	2008-09-05 09:32:47.000000000 -0500
+++ speech-tools-2.0.95~beta/utils/EST_cutils.c	2010-07-17 09:29:38.000000000 -0500
@@ -54,12 +54,19 @@
 #ifdef ESTLIBDIRC
 #    define ESTLIBDIR STRINGIZE(ESTLIBDIRC)
 #endif
-
 #ifndef ESTLIBDIR
 #define ESTLIBDIR "/usr/local/lib/speech_tools"
 #endif
 
+#ifdef ESTDATADIRC
+#    define ESTDATADIR STRINGIZE(ESTDATADIRC)
+#endif
+#ifndef ESTDATADIR
+#define ESTDATADIR "/usr/local/share/speech_tools"
+#endif
+
 const char * const est_libdir = ESTLIBDIR;
+const char * const est_datadir = ESTDATADIR;
 
 const char * const est_ostype = STRINGIZE(ESTOSTYPE);
 
